app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanelContent.tsx

```diff
@@ -10,21 +10,21 @@ import {
 import { ConnectionPendingIcon } from '../icons/ConnectionPendingIcon';
 import { NoDataIcon } from '../icons/NoDataIcon';
 import { DuplicateServerIcon } from '../icons/DuplicateServerIcon';
-import { InProductHelpAction, InProductHelpActionType } from '../InProductHelpDrawer/InProductHelpAction';
+import { InProductHelpAction, InProductHelpActionButtonAppearance, InProductHelpActionType } from '../InProductHelpDrawer/InProductHelpAction';
 import { InProductHelpIds } from '../InProductHelpDrawer/InProductHelpIds';
 
 type NotConnectedStateProps = {
 	connectedState: ConnectedState;
 	contentHeader: string,
 	contentInstructionOne: string,
 	contentInstructionTwo?: string,
-	buttonAppearance: Appearance,
+	buttonOneAppearance?: Appearance,
 	firstButtonLabel?: string,
 	secondButtonLabel?: string,
-	buttonOneOnClick(data?: any): void,
+	buttonOneOnClick?(data?: any): void,
 	buttonTwoOnClick?(data: any): void,
```

Comment:
Looks like this "buttonOne" "buttonTwo" naming was pre-existing but it would be great to update

Reply:
same as my reply above.

Reply:
`primaryActionOnClick` `secondaryActionOnClick` would be my suggestions here :)